app/views/podcast_planner/_form_date_calculator.html.erb

```diff
@@ -48,35 +45,40 @@
   <div class="mb-4" data-controller="toggle-field">
     <h3 class="fw-bold"><%= t(".header.range_select") %></h3>
     <p><%= t(".help.range_select") %></p>
-    <div class="row mb-4">
-      <div class="col-lg-6">
-        <p><%= t(".help.calculate_range") %></p>
-        <div>
-          <div class="form-floating">
-            <%= form.select :date_range_condition, end_condition_options, {}, data: {action: "toggle-field#displayOnlyThisField click#submit"} %>
-            <%= form.label t(".label.date_range_condition") %>
-          </div>
+    <div class="card mb-4">
+      <div class="card-body p-4 pb-0">
```

Comment:
I put the range in a card and adjusted the layout just a bit. All the functionality is there, I think this is a little cleaner from a UX standpoint.